src/adapters/postgres/fields-adapter.ts

```
@@ -1761,12 +1761,26 @@ export class PostgresFieldsService implements IServicelocatorfields {
             }
           });
         } else if (data.sourceDetails.source === 'table') {
+          let whereCondition = `value='${data.value}'`;
+
+          // Check if `data.value` contains multiple comma-separated values
+          if (data.value.includes(',')) {
```

ESLint

[error] Replace `','` with `","`

(prettier/prettier)